app.py

```diff
@@ -382,13 +389,14 @@ def write_data(self, body):
             return
         else:
             tracking_id = str(self.request.headers.get('Tracking-ID', "null"))
-            service = split_content(self.request.files['upload'][0]['content_type'])
+            self.metadata = self.arguments.get('metadata')[0].decode('utf-8') if self.arguments.get('metadata') else None
```

Comment:
I thought the metadata was a separate part in the body.  wouldn't that be accessed via the `files` attribute?

Reply:
Tornado had issues with two form parts. It was only picking up the actual file upload. I had to change around the way that worked completely. I tested this with `-F 'upload=@file' -F 'metatdata={"some-json": "stuff"}` and it works using this method. Using the old method, the only thing that got picked up was the upload field